### DISCUSSION TITLE
Scroll through CommitView from the log

### DISCUSSION THREAD

**Discussion by arnold-c** [NeogitOrg/neogit#1493] [Q&A]
When in the log buffer (i.e., NeogitLogView), you can hit enter on a commit to open it in a split (i.e., NeogitCommitView). Is there a way to navigate back to the log view, and when you move your cursor down to the next commit, it updates the NeogitCommitView in the split, a little like how `C-j` opens the fold of the next commit message in the log?

**Comment by CKolkey** [comment]
Yup! There's bindings for `OpenOrScrollUp` and `OpenOrScrollDown` - bound to `]c` and `[c` by default.

**Reply by arnold-c** [reply]
I'm afraid those commands are not doing quite what I was thinking of (apologies if I was unclear in the question). `[c` / `]c` is scrolling up/down within a single commit message. I was wondering if there was a command to switch to a different commit e.g., go from showing commit a to showing commit b?

**Reply by CKolkey** [reply]
Ohh, sorry, misunderstood. No, there isn't really - <c-j/k> will go up and down opening and closing the folds, but I could extend it to work for the commit view if thats open :)

**Reply by arnold-c** [reply]
Not a problem at all - I wasn't very clear in my original message and can see how it would be easily interpreted that way. If you're willing to, I think this would be a great UX improvement as I often find myself scanning commits details when coming back to projects, navigating for bisects etc., and being able to do so without needing to continuously close and reopen the commit message would be great! Should I transfer this to an issue so it can be tracked for development?

**Reply by CKolkey** [reply]
Nah, all good, it should be pretty simple. Glad you suggested it, it's definitely the right move

**Reply by arnold-c** [reply]
Just updated Neogit and the new feature is great! Thanks so much for implementing it so quickly.

**Reply by CKolkey** [reply]
Sure thing! It was a good suggestion, glad you asked :)
